README.md

```diff
@@ -1,5 +1,7 @@
 Docker Compose
 ==============
+![Docker Compose](https://github.com/docker/compose/blob/master/docs/logo.png).
```

Comment:
^^ which, I think, would have the advantage that it also works on a Forked repo :-)